src/trap.js

```diff
@@ -70,82 +81,145 @@ class Trap {
 
   // Metadata API
 
-  // `streamId` getter
+  /**
+   * `streamId` getter
+   *
+   * @returns {string}
+   */
   streamId() {
     return this._metadata.streamId;
   }
 
-  // `sessionId` getter
+  /**
+   * `sessionId` getter
+   *
+   * @returns {string}
+   */
   sessionId() {
     return this._metadata.sessionId;
   }
 
-  // Set API key header name
+  /**
+   * Set API key header name
+   *
+   * @param {string} apiKeyName
+   */
   apiKeyName(apiKeyName) {
     this._metadata.apiKeyName = apiKeyName;
   }
 
-  // Set API key value that is used to identify a client
+  /**
+   * Set API key value that is used to identify a client
```

Comment:
Client as the integrator organization/person who uses the data collection library.

Reply:
Done